crates/hdi/src/hdi.rs

```diff
@@ -189,6 +189,7 @@ impl HdiT for HostHdi {
     fn zome_info(&self, _: ()) -> ExternResult<ZomeInfo> {
         host_call::<(), ZomeInfo>(__hc__zome_info_1, ())
     }
+    #[cfg(feature = "unstable-hdk-functions")]
```

Comment:
```suggestion
    #[cfg(feature = "unstable-functions")]
```

It seems unnecessary to me to include the crate name in the feature

Reply:
Weren't we going to distinguish between hdk and hdi functions that may be unstable?

Reply:
Even "unstable hdk" / "unstable hdi" is too coarse grained anyway. For instance `is_same_agent` and `get_agent_key_lineage` are in the HDI and HDK respectively, but they are both part of the ability to handle agent key updates, so both should probably be behind a feature flag for key updates.